views/partials/inventory/product-order/detail.jade

```diff
@@ -3,38 +3,43 @@
     include ./breadcrumb.jade
   .col-xs-12(ng-switch="detailVM.item.status")
     div(ng-switch-when="LOADED")
-      .panel.panel-default.z-depth-1
+      .panel.panel-default
         .panel-heading
-          .h4
-            strong
-              time(title="{{ detailVM.item.requestedStartDate | amDateFormat: 'dddd, MMMM Do YYYY, h:mm a' }}") {{ detailVM.item.requestedStartDate | amCalendar }}
```

Comment:
This is incorrect, you are using the formatted date as title attribute while you are displaying the simple mm/dd/yyyy I guess they should be swapped.